subset/security/tlstest/src/main/java/Main.java

```diff
@@ -0,0 +1,25 @@
+public class Main {
+
+  public static void main(String[] args) {
+    String ipAddress = "127.0.0.1";
+
+    try {
+      if (args.length > 0) {
+        if (args[0].length() > 0) {
```

Comment:
no need to check the length of the string itself

Reply:
I dont follow, this is what you asked for instead of if (args != null)
or do you mean:

if(args[0].length() > 0)

should be :

if(args[0])